src/components/sidebar.tsx

```diff
@@ -1,24 +1,85 @@
-import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
+import logo from "../assets/images/logo.png";
+
+import user from "../assets/icons/user.svg";
+import projects from "../assets/icons/projects.svg";
+import schedule from "../assets/icons/schedule.svg";
+import chat from "../assets/icons/chat.svg";
 
 export function Sidebar() {
   return (
-    <div className="bg-gray-200 w-64">
-      <nav>
-        <ul>
-          <li>
-            <Link to="/users">Equipe</Link>
-          </li>
-          <li>
-            <Link to="/projects">Projetos</Link>
-          </li>
-          <li>
-            <Link to="/schedule">Calendário</Link>
-          </li>
-          <li>
-            <Link to="/chat">Chat</Link>
-          </li>
+    <div className="w-48 bg-gradient-to-b from-sideBar-Top from-0% to-sideBar-Bottom to-35% shadow-3xl flex flex-col justify-between text-center">
+      <div className="flex flex-col items-center mt-6 text-white gap-1">
+        <img
+          src="https://avatars.githubusercontent.com/u/102186472?v=4"
+          alt="userPhoto"
+          className="rounded-full h-24 w-24 border-white border-2"
+        />
+        <div>
+          <div>Cristiano Silva</div>
+          <div>(Admin)</div>
+        </div>
+      </div>
+      <nav className="m-4">
+        <ul className="text-white">
+          <NavLink
+            className="flex align-middle items-center mb-5"
+            to="/users"
+            style={({ isActive }) => ({
+              border: isActive ? "2px solid #942EA5" : "2px solid transparent",
+              borderRadius: isActive ? "10px" : "10px",
+              padding: isActive ? "10px" : "10px",
+              transition: "border-color 0.2s ease-in-out",
+            })}
+          >
+            <img src={user} alt="" className="h-8 w-1/2" />
+            Equipe
+          </NavLink>
+          <NavLink
+            className="flex align-middle items-center mb-5"
```

Comment:
As all the NavLinks are using the same classes, you could put them all into a single const and pass it over to them. In this way, if any change in the style needs to be applied, you can change it in a single place, instead of all the NavLinks components

Reply:
i took you suggestion and i did a const with a string to replace that, saving some code lines in the process